### ISSUE TITLE
PDF files without a filename cannot be split

### ISSUE THREAD

**Issue by awalker4** [Unstructured-IO/unstructured-js-client#100]
See [this comment](https://github.com/Unstructured-IO/unstructured-js-client/issues/20#issuecomment-2199765184). If a pdf file does not have `.pdf` in the filename, we return the message `Given file is not a PDF. Continuing without splitting.`. The issue is that loadPdf [here](https://github.com/Unstructured-IO/unstructured-js-client/blob/0fdd5e9c8defbc629d729eb33b2b53d2b2fbb213/src/hooks/custom/utils/pdf.ts#L101) should not return as soon as the file extension check fails. Let's try to load the file in pdf-lib, and only return if that fails.

**Comment by Ziao** [comment]
Same thing happens when passing in a blob, which obviously has no filename. Shouldn't the library simply use the mimetype or trust that the passed contentType is correct?

**Comment by awalker4** [comment]
Yes, apologies for the delay here! This will be addressed in #115
